### PR TITLE
add path for users except student

### PR DESCRIPTION
**Pull request by tsubasa9647**

### CODE REVIEW

File: infrastructure/router.go [hunk 1]
```diff
@@ -112,13 +123,23 @@ func Router(controllers *controllers.RootController, handlers *handler.RootHandl
 		adminEndPoint.Use(handlers.AuthHandler.AdminAuthHandler())
 		adminEndPoint.GET("/me", handlers.MeHandler.AdminMeHandler())
 
+		adminEndPoint.POST("/student", func(c *gin.Context) { controllers.StudentsController.Create(c) })
```
Comment (arabian9ts) [inline infrastructure/router.go]: `students`かな

File: infrastructure/router.go [hunk 2]
```diff
@@ -112,13 +123,23 @@ func Router(controllers *controllers.RootController, handlers *handler.RootHandl
...
+		adminEndPoint.GET("/students/:student_id", func(c *gin.Context) { controllers.StudentsController.Show(c) })
+		adminEndPoint.PUT("/students/:student_id", func(c *gin.Context) { controllers.StudentsController.Update(c) })
+
+		adminEndPoint.POST("/assistant", func(c *gin.Context) { controllers.AssistantsController.Create(c) })
+		adminEndPoint.GET("/assistants/:assistant_id", func(c *gin.Context) { controllers.AssistantsController.Show(c) })
+		adminEndPoint.PUT("/assistants/:assistant_id", func(c *gin.Context) { controllers.AssistantsController.Update(c) })
+
+		adminEndPoint.POST("/teacher", func(c *gin.Context) { controllers.TeachersController.Create(c) })
```
Comment (arabian9ts) [inline infrastructure/router.go]: `teachers`

File: infrastructure/router.go [hunk 3]
```diff
@@ -112,13 +123,23 @@ func Router(controllers *controllers.RootController, handlers *handler.RootHandl
 		adminEndPoint.Use(handlers.AuthHandler.AdminAuthHandler())
 		adminEndPoint.GET("/me", handlers.MeHandler.AdminMeHandler())
 
+		adminEndPoint.POST("/student", func(c *gin.Context) { controllers.StudentsController.Create(c) })
+		adminEndPoint.GET("/students/:student_id", func(c *gin.Context) { controllers.StudentsController.Show(c) })
+		adminEndPoint.PUT("/students/:student_id", func(c *gin.Context) { controllers.StudentsController.Update(c) })
+
+		adminEndPoint.POST("/assistant", func(c *gin.Context) { controllers.AssistantsController.Create(c) })
```
Comment (arabian9ts) [inline infrastructure/router.go]: `assistants`

**Review by arabian9ts — CHANGES_REQUESTED** [review]

File: infrastructure/router.go (hunk 1 above)
Comment (tsubasa9647) [reply infrastructure/router.go]: d76b0f7

File: infrastructure/router.go (hunk 2 above)
Comment (tsubasa9647) [reply infrastructure/router.go]: d76b0f7

File: infrastructure/router.go (hunk 3 above)
Comment (tsubasa9647) [reply infrastructure/router.go]: d76b0f7

**Review by arabian9ts — APPROVED** [review]

**Pull request merged**